File: openapi-generator/dart-jaguar/lib/model/inline_object20.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'inline_object20.jser.dart';

class InlineObject20 {
   /* Group ID to query. */
  @Alias('groupId')
  final int groupId;
   /* List of sensor IDs to query. */
  @Alias('sensors')
  final List<int> sensors;
  

  InlineObject20(
      

{
    
     this.groupId = null,  
     this.sensors = const [] 
    }
  );

  @override
  String toString() {
    return 'InlineObject20[groupId=$groupId, sensors=$sensors, ]';
  }
}

@GenSerializer(nullableFields: true)
class InlineObject20Serializer extends Serializer<InlineObject20> with _$InlineObject20Serializer {

}

File: openapi-generator/scalaz/src/main/scala/org/openapitools/client/api/InlineObject20.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import InlineObject20._

case class InlineObject20 (
  /* Group ID to query. */
  groupId: Long,
/* List of sensor IDs to query. */
  sensors: List[Long])

object InlineObject20 {
  import DateTimeCodecs._

  implicit val InlineObject20CodecJson: CodecJson[InlineObject20] = CodecJson.derive[InlineObject20]
  implicit val InlineObject20Decoder: EntityDecoder[InlineObject20] = jsonOf[InlineObject20]
  implicit val InlineObject20Encoder: EntityEncoder[InlineObject20] = jsonEncoderOf[InlineObject20]
}
